easy-ui-react/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@easypost/easy-ui",
   "version": "1.0.0",
   "scripts": {
+    "lint": "eslint src --ext .ts,.tsx && stylelint 'src/**/*.{css,scss}' && npx prettier --check ./src",
```

Comment:
you know what, i forgot that our eslint is configured to check prettier as well so we can remove the prettier check line :)

then i think we can optimize this lint script with the following:

<img width="906" alt="image" src="https://user-images.githubusercontent.com/752942/222489884-82013fba-46b6-48d5-9723-3db7b5b6a315.png">

that way they can take advantage of being run in parallel (they don't depend on each other). `run-p` uses the `npm-run-all` package that's installed at the root to run tasks in parallel